### PR TITLE
Making it so that styles are added to the head of the document

### PR DESCRIPTION
**Pull request by gauntface**
@addyosmani @paullewis thoughts on this?

### CODE REVIEW

File: src/scripts/controller/ActivityController.js [hunk 1]
```diff
@@ -39,7 +58,18 @@ export default class ActivityController {
   }
 
   onFinish() {
-    console.log('onFinish: ', this.path);
+    console.log('onFinish');
```
Comment (paullewis) [inline src/scripts/controller/ActivityController.js:61]: Probs want to lose this, no?

**Comment by paullewis** [comment]
LGTM!

File: src/scripts/controller/ActivityController.js (hunk 1 above)
Comment (addyosmani) [inline src/scripts/controller/ActivityController.js:61]: Was going to drop this line in Matt's branch but looks like this is consistent with the console logging for `onUpdate`, `onStart` and so on. Yay consistency :tada:

**Comment by addyosmani** [comment]
LGTM2

**Pull request merged**